src/App.tsx

```diff
@@ -10,9 +11,32 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/">
+              <Route
+                index
+                element={<h1 className="title">Home Page</h1>}
+              />
+              <Route
+                path="/home"
+                element={<Navigate to="/" replace />}
```

Comment:
Not sure you need indentations as you have only 2 attributes (unless over 80 characters of course)

Reply:
strongly agree